packages/markdown-editor/src/MarkdownEditor.js

```diff
@@ -86,6 +95,16 @@ function prepareLink({ type, itemId, addon }) {
   }
 }
 
+function SpecialCharactersExtended(editor) {
+  editor.plugins.get('SpecialCharacters').addItems('Mathematical', [
+    { title: 'smiley face', character: '😊' },
+    { title: 'rocket', character: '🚀' },
+    { title: 'wind blowing face', character: '🌬️' },
+    { title: 'floppy disk', character: '💾' },
+    { title: 'heart', character: '❤️' },
+  ])
+}
+
```

Comment:
Could you add the emojis has a new category please ? Like in this example: https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html#adding-a-new-category

Also reformatting this [json](https://raw.githubusercontent.com/omnidan/node-emoji/master/lib/emoji.json) (and importing the result in the `addItems` call) would allow adding a lot more emojis at once.

Finally is there a way to replace the toolbar **Ω** icon by a smiley ? It would make the emojis a lot more discoverable for new users.

Reply:
I have added the emojis in "https://raw.githubusercontent.com/omnidan/node-emoji/master/lib/emoji.json" as a new category "emojis".
The "Ω" icon in the toolbar is default by ckeditor and the only way to change it is to edit the 
"node_modules/@ckeditor/ckeditor5-special-characters/src/specialcharacters.js" file.
The change would be reverted everytime node_modules are installed.